CMakeLists.txt

```diff
@@ -23,11 +23,14 @@ etn_target(shared ${PROJECT_NAME}
         *.h
     USES_PRIVATE
         cxxtools
-        protobuf        
+        protobuf
 )
 
+# Remove protobuf warning
+target_compile_options(${PROJECT_NAME} PUBLIC "-Wno-useless-cast")
```

Comment:
Could be add in the FLAGS  section of etn_target. See https://github.com/42ity/fty-cmake/blob/master/README.md

Reply:
done